examples/landing-zone-v2/tier4/compute-address/Kptfile

```diff
@@ -0,0 +1,16 @@
+apiVersion: kpt.dev/v1
+kind: Kptfile
+metadata:
+  name: example-compute-address
+  annotations:
+    config.kubernetes.io/local-config: "true"
+info:
+  description: |
+    Landing zone v2 tier3 resource example.
```

Comment:
line 9: tier3 should be replaced with tier4 (same thing on line 12)

Reply:
Fixed. Good catch. I copied and pasted and forgot to update these.